source/includes/shopper_activity/_cart_activity.md

```diff
@@ -176,11 +176,13 @@ end
           <tbody>
             <tr>
               <td><code>product_id</code></td>
-              <td>Required. A unique identifier for the product from the <code>provider</code>.</td>
+              <td>Optional. A unique identifier for the product from the <code>provider</code>.
+              We recommend using this field whenever possible to allow for product level segmentation.
+              (Note: This field is used by segmentation.)</td>
```

Comment:
I think we left these fields required for cart for now.

Reply:
@will-in-wi new commit has been added to fix this issue.  Ready for your eyes again.